core/src/i18n/en.yaml

```diff
@@ -108,12 +108,16 @@ applications:
     namespace: Namespace
   messages:
     app-created: Application {{applicationName}} created.
+    binding-create-failed: Failed to create the Binding.
+    binding-created: Binding created.
     binding-deleted: Binding deleted.
     binding-not-found: No bindings.
     cannot-create: Cannot create the Application.
+    connect-disabled: Connecting an Application is disabled for SKR runtime.
```

Comment:
```suggestion
    connect-disabled: Connecting the Application is disabled for SAP Kyma Runtime.
```

Reply:
I'd either go with `SAP Kyma Runtime` or `SKR` because the SKR abbreviation already has the word "runtime" included.

Reply:
I'd also be consistent with the articles. In previous messages, you use definite articles (`the` Bindings, `the` Application), so either change all of them to indefinite articles, or to definite ones. Let's discuss 🙂

Reply:
I thought `the` is ok for one-and-only application we were working on and something went wrong (thus `the Application`, `that Application`), and `a` is good for just _any_ application -it's a general rule, so it's not about any particular app.

Reply:
Yes, you are right 🙂 That is why I also thought it would be better to use an indefinite article in all the cases above (so: `a` Binding, `an` Application). I'll adjust the suggestions then!

Reply:
Ok, I didn't realize the subtle differences between these messages. Thanks for clarifying!